src/release/check.rs

```diff
@@ -125,12 +125,12 @@ fn next_(
             next = "21.04";
 
             ReleaseStatus {
-                build:   if development { release_check(next) } else { BuildStatus::Blacklisted },
+                build: if development { release_check(next) } else { BuildStatus::Blacklisted },
                 current: "20.10",
-                is_lts:  false,
+                is_lts: false,
```

Comment:
These don't seem right: it's *removing* the field alignment, whereas everything else *adds* field alignment. What's causing fmt to do that?

Reply:
Good catch. Rustfmt bug? Will do some digging

Reply:
if the difference in field widths is too great, this formatting rule doesn't apply. this threshold is configured to 30 characters. the max diff of field widths in this struct is greater than 30, so this is correct

the rule itself is unstable